traffic_portal/app/src/app.js

```diff
@@ -339,8 +332,6 @@ var trafficPortal = angular.module('trafficPortal', [
         require('./common/modules/table/cacheGroupServers').name,
         require('./common/modules/table/cacheGroupStaticDnsEntries').name,
         require('./common/modules/table/capabilities').name,
-        require('./common/modules/table/capabilityEndpoints').name,
```

Comment:
i don't think there's any reason to remove these views. capability endpoints and capability users are still relevant views. i think you should just leave them for future use.

Reply:
actually, i put them there even though capabilities are not in use yet and they are not accessible via a menu but you can still get to them by going to:

https://tp.domain.net/#!/capabilities/auth/users
https://tp.domain.net/#!/capabilities/auth/endpoints

Reply:
It's an inaccessible state through the UI; if you want to see those you have to already know they exist. Clicking on the table won't do anything anymore - and to get there you have to use direct URL navigation. I can leave them in but at this point they're something that's been un-linked from something that itself was just un-linked from a view that's never been linked at all (afaik).

Reply:
yeah, if you leave them in, i can link them via a context menu on the caps table at a later date since you removed the page where they were previously linked from (the edit capability page)